modules/apps/jaggery/emm/src/emm/units/user-update/user-update.hbs

```diff
@@ -52,7 +52,7 @@
                         </div>
                     </div>
                     <br>
-                    <button id="add-user-btn" class="wr-btn">&nbsp;&nbsp;&nbsp;&nbsp;Edit User&nbsp;&nbsp;&nbsp;&nbsp;</button>
+                    <button id="add-user-btn" class="wr-btn">&nbsp;&nbsp;&nbsp;&nbsp;Save User&nbsp;&nbsp;&nbsp;&nbsp;</button>
```

Comment:
Noted.
And this can be found in all the buttons in EMM User Interface.This will be fixed in 2.0.1 patch and it currently in progress under the Jira[1].
[1]. https://wso2.org/jira/browse/EMM-1233